TopPlayer.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>

```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
    <title>Go Gaming</title>
    <link href="TopPlayer.css" type="text/css" rel="stylesheet">
</head>
<body>
     <header>
                <marquee><h1>Go Gaming</h1></marquee>
        </header>
        
    
        <nav>
        <a href="http://127.0.0.1:5500/Description.html">&infin;Description</a>
        <a href="http://127.0.0.1:5500/place.html">&infin;Places</a>
        <a href="http://127.0.0.1:5500/contest.html">&infin;Contests</a>
        <a href="http://127.0.0.1:5500/topswimmer.html">&infin;Top Player</a>
        </nav>
      
        <div class="table">
                <h2 style="text-align: center;">Top Player In Indonesian</h2>
        </div>

        <div id="content1">
            <h2>BTR Zuxxy</h2>
            <img src="BTR Zuxxy.jpg" alt="" width="70%" height="60%">
            <p>BTR Zuxxy or original origin, Made Bagas is a sibling and twin of BTR Luxxy. Zuxxy, in an interview at PMSC yesterday, said that he was born "8 minutes" earlier than his younger brother, Luxxy. Zuxxy's position in the game as Co-Leader and Observer who supports the player's movements.</p>
       </div>
      
        <div id="content2">
            <h2>EVR389m0zaASLAN</h2>
            <img src="Benny Moza.jpg" alt="pic not here" width="65%" height="55%">
            <p>' Bennymoza 'this is the journey of his story to become a professional game player. Moza is a Point Blank player who has a support role, in his team's line-up. The man born October 21, 1990 was born with the real name Benny Setiawan.</p>        
        </div>
            

            <div id="footer1">
           <h5>BlaBlaBlaGaming.com</h5>
       </div>
</body>
</html>